packages/ses/scripts/bundle.js

```diff
@@ -36,4 +37,12 @@ const write = async (target, content) => {
   await write('dist/lockdown.mjs', bundle);
   await write('dist/lockdown.umd.js', bundle);
   await write('dist/lockdown.umd.min.js', terse);
-})();
+};
+
+main().then(
+  () => process.exit(0),
+  err => {
+    console.error('Error running main:', err);
+    process.exit(1);
+  },
```

Comment:
Might consider using `process.exitCode = 1` so any work that might accidentally linger after main settles is observable.

https://nodejs.org/api/process.html#processexitcode_1